mlforecast/distributed/forecast.py

```diff
@@ -207,6 +207,7 @@ def predict(
         dynamic_dfs: Optional[List[pd.DataFrame]] = None,
         before_predict_callback: Optional[Callable] = None,
         after_predict_callback: Optional[Callable] = None,
+        new_data: Optional[pd.DataFrame] = None,
```

Comment:
This should probably be dask dataframe. But maybe it isn't worth having this in the distributed interface. I guess it's more common to do distributed training and then save the trained model for predicting smaller datasets locally, we should probably have a `to_local` method instead so that after training users can convert the distributed forecast to a local one and then use that one with `new_data`. WDYT?

Reply:
I agree! Maybe we can leave the possibility to use the new feature in a distributed way (I already corrected the typing) and open an issue to include the `to_local` method. WDYT?